agent/README.md

```diff
@@ -59,7 +59,7 @@ The following options are available:
   Teamscale uses this to map the coverage to the corresponding source code. Thus, this must be the exact code commit 
   from the VCS that was deployed. For an alternative see `teamscale-commit-manifest-jar`.
 - `azure-url`: a HTTPS URL to an azure file storage. Must be in the following format: 
-  https://<account>.file.core.windows.net/<share>/(<path>). The <path> is optional; note, that in the case that the given
+  https://<account>.file.core.windows.net/\<share\>/(\<path\>). The \<path\> is optional; note, that in the case that the given
```

Comment:
Escape <account> as well. Or maybe just make the entire URL `code`?

Reply:
Dunno how I overlooked that one :man_facepalming:

Reply:
C'mon, admit that you just wanted to try me 😂